account_invoice_reminder/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,4 @@
+* `Coop IT Easy SCRLfs <https://coopiteasy.be>`_:
+
+  * Houssine BAKKALI <houssine.bakkali@gmail.com>
+  * Carmen Bianca Bakker <carmen@coopiteasy.be>
```

Comment:
the current policy (as discussed [here](https://github.com/coopiteasy/cie-timesheet/pull/8#discussion_r701952779)) is to only list the names without email addresses. please regenerate the readme aftewards. 😉

Reply:
fixed and regenerated